config/default.json

```diff
@@ -20,5 +20,13 @@
         }
       }
     }
+  },
+  "bom": {
```

Comment:
Why is this needed?

Reply:
You can configure the properties that could be ignored for alignment https://github.com/snowdrop/snowdrop-automation-client/pull/101/files#diff-bb725bdb4483849adf186a500d56c277R11. This was always there.

Reply:
OK, just wondering why hibernate is ignored by default… and actually, with the current release process I'd argue that, by default, we shouldn't ignore any properties and update all of them.

Reply:
Sure, we could remove all. I think it was there because there was always a big divergence between Hibernate versions in Spring Boot and prod.

Reply:
Possible, yes but it should be removed by default, I think.

Reply:
I can remove it here, since we've reformatted the bom and changing the way we do the releases.

But just to clarify, I don't think there is a default in this project (I know the config file is called default, but that's just how this lib works). We clone and run the project whenever needed, so all non-private config should be stored in GitHub I think.